db/migrate/20240402043253_create_arm_tables.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+class CreateArmTables < ActiveRecord::Migration[7.1]
+  # rubocop:disable Metrics/MethodLength
+  def change
+    create_table :accredited_representatives, id: false do |t|
+      t.string :number, null: false
+      t.string :poa_code
+      t.string :first_name
+      t.string :middle_initial
+      t.string :last_name
+      t.string :full_name
+      t.string :email
+      t.string :phone
+      t.string :address_type
+      t.string :address_line1
+      t.string :address_line2
+      t.string :address_line3
+      t.string :city
+      t.string :country_code_iso3
+      t.string :country_name
+      t.string :county_name
+      t.string :county_code
+      t.string :international_postal_code
+      t.string :province
+      t.string :state_code
+      t.string :zip_code
+      t.string :zip_suffix
+      t.jsonb :raw_address
+      t.float :lat
+      t.float :long
+      t.geography :location, limit: { srid: 4326, type: 'st_point', geographic: true }
+      t.timestamps
+    end
+    # rubocop:enable Metrics/MethodLength
+    safety_assured do
+      execute 'ALTER TABLE accredited_representatives ADD PRIMARY KEY (number);' # rubocop:disable Rails/ReversibleMigration
+    end
+    add_index :accredited_representatives, :full_name
+    add_index :accredited_representatives, :location, using: :gist
+    add_index :accredited_representatives, %i[number first_name last_name], unique: true, name: 'index_rep_group'
```

Comment:
we want the uniqueness constraint on just the registration number to avoid non-unique primary keys like the veteran_x tables

Reply:
Yes, you're right. I'll add it and remove this:

`add_index :accredited_representatives, %i[number first_name last_name], unique: true, name: 'index_rep_group'`